File: RxRetrofitApp/japp/src/main/java/com/woowrale/rxretrofit/japp/ui/main/MainView.java
package com.woowrale.rxretrofit.japp.ui.main;

import com.woowrale.rxretrofit.japp.data.model.Crypto;
import com.woowrale.rxretrofit.japp.ui.base.BasePresenter;

import java.util.List;

public interface MainView extends BasePresenter.View {

    void showListView(List<Crypto.Market> marketList);

}

File: RxRetrofitApp/kapp/src/main/java/com/woowrale/rxretrofit/kapp/ui/main/MainView.kt
package com.woowrale.rxretrofit.kapp.ui.main

import com.woowrale.rxretrofit.kapp.data.model.Market
import com.woowrale.rxretrofit.kapp.ui.base.BasePresenter

interface MainView: BasePresenter.View {
    fun setRecyclerViewAdapterData(marketList: List<Market>)
}